### ISSUE TITLE
Button: accessibilty violation.

### ISSUE THREAD

**Issue by brunano21** [ibm-js/deliteful#642]
With current `Button`'s template, when we have a icon-only button, an a11y violation is present.
In fact, as stated [here](https://www.w3.org/TR/wai-aria/states_and_properties#aria-labelledby), the `aria-labelledby` should be used only **if the label text is visible on screen**. It label text is not visible, `aria-label` should be used instead.
